tests/pdctl/hot/hot_test.go

```diff
@@ -117,20 +117,38 @@ func (s *hotTestSuite) TestHot(c *C) {
 			c.Assert(hotRegion.AsLeader[hotStoreID].Stats[count-1].RegionID, Equals, hotRegionID)
 		}
 	}
-	reportIntervals := []uint64{
-		statistics.HotRegionReportMinInterval,
-		statistics.HotRegionReportMinInterval + 1,
-		statistics.RegionHeartBeatReportInterval,
-		statistics.RegionHeartBeatReportInterval + 1,
-		statistics.RegionHeartBeatReportInterval * 2,
-		statistics.RegionHeartBeatReportInterval*2 + 1,
+	reportIntervalsFunc := []func(reportInterval uint64) uint64{
+		func(reportInterval uint64) uint64 {
+			return statistics.HotRegionReportMinInterval
+		},
+		func(reportInterval uint64) uint64 {
+			return statistics.HotRegionReportMinInterval + 1
+		},
+		func(reportInterval uint64) uint64 {
+			return reportInterval
+		},
+		func(reportInterval uint64) uint64 {
+			return reportInterval + 1
+		},
+		func(reportInterval uint64) uint64 {
+			return reportInterval * 2
+		},
+		func(reportInterval uint64) uint64 {
+			return reportInterval*2 + 1
+		},
 	}
-	for _, reportInterval := range reportIntervals {
-		hotReadRegionID, hotWriteRegionID := reportInterval, reportInterval+100
-		pdctl.MustPutRegion(c, cluster, hotReadRegionID, hotStoreID, []byte("b"), []byte("c"), core.SetReadBytes(1000000000), core.SetReportInterval(reportInterval))
-		pdctl.MustPutRegion(c, cluster, hotWriteRegionID, hotStoreID, []byte("c"), []byte("d"), core.SetWrittenBytes(1000000000), core.SetReportInterval(reportInterval))
+	regionIDCounter := uint64(1)
+	for _, reportIntervalFunc := range reportIntervalsFunc {
+		hotReadRegionID := regionIDCounter
+		regionIDCounter++
+		hotWriteRegionID := regionIDCounter
+		regionIDCounter++
+		readReportInterval := reportIntervalFunc(statistics.ReadReportInterval)
+		writeReadReportInterval := reportIntervalFunc(statistics.WriteReportInterval)
+		pdctl.MustPutRegion(c, cluster, hotReadRegionID, hotStoreID, []byte("b"), []byte("c"), core.SetReadBytes(1000000000), core.SetReportInterval(readReportInterval))
+		pdctl.MustPutRegion(c, cluster, hotWriteRegionID, hotStoreID, []byte("c"), []byte("d"), core.SetWrittenBytes(1000000000), core.SetReportInterval(writeReadReportInterval))
 		time.Sleep(5000 * time.Millisecond)
-		if reportInterval >= statistics.RegionHeartBeatReportInterval {
+		if readReportInterval >= statistics.ReadReportInterval || writeReadReportInterval >= statistics.WriteReportInterval {
```

Comment:
Maybe it makes more sense to split it into two foreachs for reading and writing? I'm not sure if `if readReportInterval >= statistics.ReadReportInterval || writeReadReportInterval >= statistics.WriteReportInterval` is the same as the previous test Equivalent